### DISCUSSION TITLE
Implementation

### DISCUSSION THREAD

**Discussion by Thewsthews** [community/community#144203] [Programming Help] [answered]
### Body

After code compilation, im getting this error:
`impl Component for MyComponent {
   |                                - while parsing this item list starting here
...`

Below is the code sample: 
`impl Component for MyComponent {
    type Message = Msg;
    type Properties = ();

    fn create(_ctx: &Context<Self>) -> Self{
        Self{
            count: 0,
            input_value: String::new(),
            background_color: "white".to_string(),
        }
    }` 

Any way to go about it?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Nathanael349** [comment, marked as answer]
I think that your code is missing a closing brace, }, for the impl block. You can try adding it after the create function:

impl Component for MyComponent {
    type Message = Msg;
    type Properties = ();

    fn create(_ctx: &Context<Self>) -> Self {
        Self {
            count: 0,
            input_value: String::new(),
            background_color: "white".to_string(),
        }
    }
} // Add this closing brace

This should fix your problem!
